app/routes/login-callback/route.tsx

```diff
@@ -0,0 +1,37 @@
+import { LoaderFunctionArgs, redirect } from '@remix-run/node';
+import { commitSession, getSession, initializeEcomApiForRequest } from '~/lib/ecom/session';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+    const session = await getSession(request.headers.get('Cookie'));
+    const storedOauthData = session.get('oAuthData');
+
+    if (storedOauthData === undefined) {
+        return redirect('/login');
```

Comment:
I know that Codux won't show this route, but let's still double check that everything works fine in the define app mode, as it doesn't support node-specific code.